app/models/user.rb

```diff
@@ -27,4 +27,9 @@ def assign_role
       add_role user_type if user_type.present?
     end
   end
+  def full_name_with_email
+    if personal_info.present?
```

Comment:
@KaranBamaniya-BTC : What about if personal info not present, it will not show user